common/data/src/main/java/org/thingsboard/server/common/data/page/TimePageLink.java

```diff
@@ -76,20 +76,4 @@ public Sort toSort(SortOrder sortOrder, Map<String,String> columnMap) {
         }
```

Comment:
Do you need `public Sort toSort(SortOrder sortOrder, Map<String,String> columnMap)`?